### DISCUSSION TITLE
Problems with simulations in pregnancy

### DISCUSSION THREAD

**Discussion by mmalsmadi** [Open-Systems-Pharmacology/Forum#816] [Questions & Problems] [answered]
Hello,

I tried the steps for simulations in pregnancy as discussed in issue#523, and the simulation ran but no profiles were generated

please find attached the PK-SIM
[Ritonavir & Lopinavir_Pregnancy.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/5050631/Ritonavir.Lopinavir_Pregnancy.zip)

**Comment by AndreDlm** [comment]
Hi @mmalsmadi,

hmm that's interesting. When I simulate the first 12h only, I get a result for ritonavir but not for lopinavir. 
![Untitled](https://user-images.githubusercontent.com/26541603/89885281-1ecfcb80-dbcb-11ea-9c9f-c7fb761a9f9d.png)

Looking at the simulated fraction dissolved in the GI tract, it seems that lopinavir is not dissolving at all, so I assume something went wrong with the dissolution module. 
Could you maybe share the pkml-file or MoBi-file of the model (prior to import in PK-Sim)? This will be more informative and may give us an idea what went wrong here.
Thanks!

André

**Comment by mmalsmadi** [comment]
Hello Andre

please find the individual simulations in PKSim and MoBi


[Pregnant 400mg LPV_100mg RTV BID 14Days MoBiEdited.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/5056600/Pregnant.400mg.LPV_100mg.RTV.BID.14Days.MoBiEdited.zip)


Many Thanks,
Motasem

**Comment by AndreDlm** [comment, marked as answer]
Ok as suspected the solubility of lopinavir is the issue here. Specifically, the building block templates for the pregnancy model (spatial structure/passive transports) have been created with an older version of MoBi (version 6 as described [here](https://github.com/Open-Systems-Pharmacology/Pregnancy-Models#version-information)) and do not support the `pH solubility Table` used for lopinavir.
When using this old spatial structure template with this `SolubilityTable`, the parameter `intestinal solubility` is set everywhere to 0 which explains why lopinavir does not dissolve in your model.

This parameter is defined by an equation and can be found under `SpatialStructure | Lumen | <name of GI segment> | MoleculeProperties`. In newer versions, the equation for this parameter looks as follows:
![Annotation 2020-08-12 094835](https://user-images.githubusercontent.com/26541603/89989280-07531a00-dc81-11ea-86d3-295aa3b34e87.png)

In the pregnancy spatial structure this parameter is defined as follows:
![Annotation 2020-08-12 094943](https://user-images.githubusercontent.com/26541603/89989385-2baef680-dc81-11ea-9e3f-b4de77f85718.png)
In this equation, the parameter `SolubilityTable` is missing.

I see three ways to solve this:

- you can update the old pregnancy spatial structure by adding the parameter `SolubilityTable` in the equation above (and potentially also additional parameters that may be missing)
- or, if you have a constant solubility, you could just overwrite the solubility equation in MoBi with your measured value
- or, if the solubility is constant, you can go to the simulation in MoBi and manually set the solubility in each intestinal segment to the measured value. You find the solubility in each intestinal segment under `Organism | Lumen | <name of GI segment> | <Drug Name>`, e.g. here shown for the caecum:
![Annotation 2020-08-12 100357](https://user-images.githubusercontent.com/26541603/89990986-6f0a6480-dc83-11ea-9b52-e23eb758ae6e.png)


I hope this helps!

Good luck,
André

**Comment by mmalsmadi** [comment]
Hello 


I tried to enter the solubility values in MoBi, but I got the following error message



![LPV MoBi change solubility](https://user-images.githubusercontent.com/37537530/90001938-b2240200-dc9a-11ea-912b-907c59ab48f3.PNG)

**Comment by AndreDlm** [comment]
What did you do exactly? Using option 3 above, I was able to simulate lopinavir in MoBi and import it in PK-Sim for population simulations without any error

**Comment by mmalsmadi** [comment]
Hello,

I initially tried changing the values from: SpatialStructure | Lumen | <name of GI segment> | MoleculeProperties


Now, after I tried option 3, I got the following error message 
![MoBi Error2](https://user-images.githubusercontent.com/37537530/90333413-2843a400-dfce-11ea-8baa-341d22af9cc2.PNG)

With simulations settings as attached

![Simulations settings](https://user-images.githubusercontent.com/37537530/90333419-385b8380-dfce-11ea-9d1e-dd90d8fcfb24.PNG)
